RadixWallet/Clients/OnLedgerEntitiesClient/Helpers/OnLedgerEntitiesClient+CreateEntity.swift

```diff
@@ -448,18 +448,35 @@ extension OnLedgerEntitiesClient {
 				return nil
 			}()
 
-			let stakeClaimTokens: NonFunbileResourceWithTokens? = try await {
+			let stakeClaimTokens: NonFunbileResourceWithTokens? = try await { () -> NonFunbileResourceWithTokens? in
 				if let stakeClaimResource = stake.stakeClaimResource, stakeClaimResource.nonFungibleIdsCount > 0 {
 					guard let stakeClaimResourceDetails = resourceDetails.first(where: { $0.resourceAddress == stakeClaimResource.resourceAddress }) else {
 						assertionFailure("Did not load stake unit details")
 						return nil
 					}
-					let tokenData = try await getAccountOwnedNonFungibleTokenData(.init(
+					let tokens = try await getAccountOwnedNonFungibleTokenData(.init(
 						accountAddress: account.address,
 						resource: stakeClaimResource,
 						mode: .loadAll
 					)).tokens
-					return .init(resource: stakeClaimResourceDetails, tokens: tokenData)
+
+					return .init(
+						resource: stakeClaimResourceDetails,
+						stakeClaims: tokens.compactMap { token -> OnLedgerEntitiesClient.StakeClaim? in
+							guard let claimEpoch = token.data?.claimEpoch,
```

Comment:
I prefer:
```
guard
    let a = ..,
    let b = ...,
    b > 0
else {
...
}
```
format, but maybe I'm not entitled to talking Swift format now, since becoming a Rust dev :D

Reply:
:D, will do, looks better.